ras/ras_extended.py

```diff
@@ -206,14 +210,14 @@ def test3_lsvpd(self):
         for list_item in list:
             cmd = "lsvpd %s" % list_item
             self.run_cmd(cmd)
-        path_db = process.system_output("find /var/lib/lsvpd/ -iname vpd.db | "
-                                        "head -1", shell=True).strip()
+        path_db = self.run_cmd_out("find /var/lib/lsvpd/ -iname vpd.db | "
+                                   "head -1").strip()
         if path_db:
             copyfile_path = os.path.join(self.outputdir, 'vpd.db')
             shutil.copyfile(path_db, copyfile_path)
             self.run_cmd("lsvpd --path=%s" % copyfile_path)
-        path_tar = process.system_output("find /var/lib/lsvpd/ -iname vpd.*.gz"
-                                         " | head -1", shell=True).strip()
+        path_tar = self.run_cmd_out("find /var/lib/lsvpd/ -iname vpd.*.gz"
+                                    " | head -1").strip()
```

Comment:
Is it better for `run_cmd_out()` to return the output after a `strip()` ?

Reply:
@narasimhan-v here we can not adopt as in `test1_lsdevinfo` we are striping using a pattern if you say I can pass another argument to `run_cmd_out()` but I think earlier implementation hold good

Reply:
@PraveenPenguin `run_cmd_out()` can still `strip()`, and `test1_lsdevinfo()` can do another `strip('12345')` ?
Let me know if that works fine for you.

Reply:
@narasimhan-v sure I think it should work, let me try and accordingly update here

Reply:
@PraveenPenguin except for this clarification, I am fine with the code.

Reply:
works well , please have look